src/DeepHelloWorld/Tests/Frontend/angular/controllers/SayHelloController.spec.js

```diff
@@ -18,9 +19,23 @@ describe('Controllers', function() {
   describe('SayHelloController', function() {
 
     it('Controller is implemented', function() {
-      let controller = $controller('SayHelloController', {$scope: {catchSubmit: null, name: 'test'}});
+      controller = $controller('SayHelloController', {$scope: {catchSubmit: null, name: 'test'}});
 
       expect(Object.keys(controller).length).toBe(2);
     });
+
+    it('catchSubmit() method exists and executes without errors', function() {
+      let actualResult = null;
```

Comment:
'let' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).